Jenkinsfile_CNP

```diff
@@ -94,7 +94,7 @@ withPipeline(type, product, component) {
           eval \$(./bin/variables/load-preview-environment-variables.sh ${CHANGE_ID})
           ./bin/add-roles.sh
           ./bin/pull-latest-general-app-ccd-files.sh ${generalappCCDBranch}
-          ./bin/import-ccd-definition.sh "-e *-prod.json,*HNL-nonprod.json,*CUI-nonprod.json,*-shuttered.json" ${ccddefbranch}
+          ./bin/import-ccd-definition.sh "-e *-prod.json,*HNL-nonprod.json,*-shuttered.json" ${ccddefbranch}
```

Comment:
require the same changes in  before('smoketest:aat') { otherwise master pipeline will fail